editoast/migrations/2024-10-24-143158_add-towed-rolling-stock-field-description/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE towed_rolling_stock
+ADD description VARCHAR(255) NOT NULL DEFAULT '';
```

Comment:
Is `description` meant to be a free field, editable if we had a towed rs editor? If so shouldn't it be a `TEXT` instead?

We can change that later if we need it ofc, feel free to disregard this comment :)

Reply:
I'm not entirely sure when we should use `VARCHAR` or `TEXT` and I feel you do know more. It's more a human readable label. For example, `name` might be something like "ABX126" and `description` might be `FREIGHT CAR FOR 126 TONS". I do not think the string will be very long. Do you feel `TEXT` is needed? I've read a bit on the subject and `VARCHAR` is probably ok. But thanks for getting that to my attention, at the very least, I learned something.

Reply:
Note: your comment might be a sign that `label` would be a better name than `description`,  what do you think?

Reply:
Alright then maybe not `description` indeed. `label` seems fine, I'm also thinking of `long_name`, your call :)

Reply:
Done. I've chosen `label`.